.travis.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# https://github.com/travis-ci/travis-rubies
```

Comment:
good point. i’ll also explain what set does and so on. i already documented the `{}` functions (they export stuff)